core/services/vrf/integration_v2_test.go

```diff
@@ -519,15 +530,36 @@ func TestVRFV2Integration_SingleConsumer_HappyPath(t *testing.T) {
 		return len(runs) == 1
 	}, cltest.WaitTimeout(t), time.Second).Should(gomega.BeTrue())
 
-	// Mine the fulfillment that was queued.
-	mine(t, requestID, subID, uni, db)
+	// Mine the fulfillments that were queued.
```

Comment:
Minor: I think at this point in the test there is still only one fulfillment, so should probably stay `fulfillment` (singular) in the comment.

Reply:
Ah yeah good catch, fixed